.github/workflows/release.yaml

```diff
@@ -144,12 +144,14 @@ jobs:
         run: |
           unset RELEASE_MINISIGN_PRIVATE_KEY
           unset RELEASE_MINISIGN_PRIVATE_KEY_PASSWORD
+      - name: Fetch Changelog
+        run: .github/actions/changelog-action/entrypoint.sh > ${{env.VERSION}}-CHANGELOG.md
```

Comment:
There is no chance this gets published anywhere somehow correct? Ie we don't need to put this in some special temp directory?

Reply:
the temp directory is the action itself which is crushed when it terminates

Reply:
I could delete the file in a subsequent test but i think its unnecessary

Reply:
Definitely unnecessary, just making sure